spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -54,8 +58,10 @@ public ProducerDestination provisionProducerDestination(String name,
 	public ConsumerDestination provisionConsumerDestination(String name, String group,
 			ExtendedConsumerProperties<PubSubConsumerProperties> properties)
 			throws ProvisioningException {
+		// Use group name as subscription name
+		// Generate anonymous random group, if one not provided
+		String subscription = !StringUtils.hasText(group) ? "anonymous." + UUID.randomUUID().toString() : group;
```

Comment:
I would like to include a `name` into the `anonymous` subscriptions.
This way it is going to be easier to tracking them on the GCP or some other place.